packages/theme/src/components/TsDocParameter.tsx

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import TsDocSignatureTitle from './TsDocSignatureTitle';
+import { HeadingLevel } from './Heading';
+import { Heading } from '../../index';
+import TsDocSignatureType from './TsDocSignatureType';
+import TsDocBlock from './TsDocBlock';
+import TsDocComment from './TsDocComment';
+
+interface Props {
+  definition: any;
+  level?: HeadingLevel;
+}
+
+function TsDocParameter({ definition, level = 1 }: Props) {
+  const nextLevel: HeadingLevel = (level + 1) as any;
+
+  return (
+    <ul>
+      {definition.signatures && definition.signatures.length > 0 && (
+        <li>
+          <ul>
+            {definition.signatures.map(signature => (
+              <li key={signature.id}>
+                <TsDocSignatureTitle definition={signature} hideName />
+              </li>
+            ))}
+          </ul>
+
+          <ul>
+            {definition.signatures.map(signature => (
+              <li key={signature.id}>
+                <TsDocBlock
+                  level={nextLevel}
+                  definition={signature}
+                  showHeader={false}
+                />
+              </li>
+            ))}
+          </ul>
+        </li>
+      )}
+
+      {definition.indexSignature && definition.indexSignature.length > 0 && (
```

Comment:
can you add a little comment that just illustrates what the thing here is? e.g.
`// { [foo: string]: Bar} `